code/controllers/subsystems/voting/poll_types.dm

```diff
@@ -25,7 +25,7 @@
 	If you are dead, or just spawned, you get only 0.6 votes
```

Comment:
XD, ya we should keep it to 0.6

Reply:
If it were up to me, ghosts would get no votes :stalinaproves: